WalletWasabi/Models/SerializableException.cs

```diff
@@ -5,7 +5,7 @@
 namespace WalletWasabi.Models
 {
 	[JsonObject(MemberSerialization.OptIn)]
-	public class SerializableException
+	public class SerializableException : IEquatable<SerializableException>
```

Comment:
If you switch from `class` to `record` you don't need to implement `Equals`, `GetHashCode` and operators (`operator ==`, `operator !=`).

https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/exploration/records#characteristics-of-records

Reply:
Very good, ACK